cities/models.py

```diff
@@ -0,0 +1,11 @@
+from sqlalchemy import Column, Integer, String
+
+from database import Base
+
+
+class DbCity(Base):
+    __tablename__ = "city"
+
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(String(255), nullable=False)
+    additional_info = Column(String(511), nullable=False)
```

Comment:
The 'additional_info' field is set as 'nullable=False', which means it is a required field. Ensure this aligns with your application's requirements. If 'additional_info' should be optional, consider setting 'nullable=True'.